.circleci/config.yml

```diff
@@ -113,56 +113,16 @@ jobs:
             echo "Your dependencies are secure enough: $SUMMARY"
             exit 0
   testE2e:
-    machine: true
-    working_directory: *WORKSPACE
+    executor: node
     steps:
       - attach_workspace:
           at: *WORKSPACE
-      - add_ssh_keys:
-          fingerprints:
-            - "65:5f:43:fd:be:48:c8:45:d5:39:ef:86:14:d6:e1:12"
-      - run:
-          name: Add GitHub to known hosts
-          command: |
-            mkdir -p ~/.ssh
-            touch ~/.ssh/known_hosts
-            ssh-keygen -R github.com
-            ssh-keyscan github.com >> ~/.ssh/known_hosts
-      - run:
-          name: checkout backend
-          command: |
-            git clone git@github.com:luniehq/lunie-backend.git
-            cd lunie-backend
-            git checkout origin/develop
-      - run: 
-          name: Use NVM
-          command: |
-            echo 'export NVM_DIR="/opt/circleci/.nvm"' >> $BASH_ENV
-            echo ' [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
-      - run:
-          name: Install node version
-          command: |
-            nvm install v10.13.0
-            nvm alias default v10.13.0
-      - run:
-          name: Install Chrome
-          command: |
-            wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
-            sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
-            sudo apt-get update
-            sudo apt-get -y install google-chrome-stable
       - run:
-          name: Run testnet
+          # needs to be done in CI to be sure it is build once we start the tests
+          name: Run tests
           command: |
-            cd lunie-backend
-            docker-compose up --build -d
-      - run:
-          name: Test
-          command: npm run serve:dist & npm run test:e2e:local
-          no_output_timeout: 120
-      - store_artifacts:
-          path: ./screenshots
-          when: on_fail
+            PR_NUMBER="$(echo $CIRCLE_PULL_REQUEST | cut -d'/' -f 7)"
+            PASSWORD=${NETWORK_PASSWORD} CHT_PWD=${ACCOUNT_PASSWORD} yarn test:e2e:browserstack --fe=https://deploy-preview-${PR_NUMBER}--lunieio.netlify.com --api=https://staging-api.lunie.io
```

Comment:
shouldn't this be checking against production API?

Reply:
ture, one last thing I overlooked